client/scripts/controllers/server/threads.ts

```diff
@@ -160,6 +154,30 @@ class ThreadsController {
     });
   }
 
+  public async setPrivacy({ threadId, privacy, readOnly }: { threadId: Number; privacy: boolean; readOnly: boolean; }) {
```

Comment:
couldn't these arguments just be specified as `{ threadId: Number; privacy: boolean; readOnly: boolean; }`?

Reply:
The typescript compiler wasn't happy with me typing the values while simultaneously destructuring. I opted for a similar, but less verbose solution.